blog/content/post/2022-01-17-kgpe_D16_status.md

```diff
@@ -0,0 +1,149 @@
+
+# To the rescue of KGPE-D16
+
+It has been a huge blow for the community believing in privacy and liberty of
+the hardware. Thus 3mdeb tried to answer on the community requests and needs to
+bring back the platform. To prevent the board from dropping (half a year before
+the 4.12 release at the end of 2019, where the deprecation has been announced)
+3mdeb has applied for funding to [NLnet Foundation](https://nlnet.nl/),
+unfortunately the project to improve the quality of board support has been
+rejected. The application can be found on [3mdeb's GitHub](https://github.com/3mdeb/kgpe-osf/blob/master/docs/nlnet-application.md)
+
+The hope was almost lost. In the meantime 3mdeb propagated the need to protect
+the libre hardware like KGPE-D16 on various events like, e.g. FOSDEM:
+
+* [FOSDEM'20](https://archive.fosdem.org/2020/schedule/event/coreboot_amd/)
+* [FOSDEM'21](https://archive.fosdem.org/2021/schedule/event/firmware_osfsoap2/)
+
+![](/img/fosdem_logo.png)
+
+And fortunately our screams have been heard... [Vikings](Vikings.net) donated
+two KGPE-D16 boards for the development and [Immunefi](https://immunefi.com/)
+has offered to sponsor the effort of bringing back the KGPE-D16 to coreboot
+main tree. For the details visit [heads issue on GitHub](https://github.com/osresearch/heads/issues/719).
+
+# KGPE-D16 revival
+
+3mdeb started the code refactoring and improving efforts in September 2021. But
+the scope was not only to bring the code quality to coreboot's current
+standards, no. We (3mdeb and the sponsors) aimed to create even more secure and
+libre platform than it was before (back then in main coreboot branch). Besides
+the coreboot requirements 3mdeb has enabled and validated TPM 2.0 and vboot on
+the platform using 8 MB and 16 MB SPI flashes (much larger than 2 MB default
+ones). 3 months later the finalized firmware has been released under
+[Dasharo](https://dasharo.com/) brand with multi-flavored pre-built binaries
+ready to use by end users. Now the platform benefits from automated tests,
+regular releases and Dasharo firmware quality. Do not hesitate and try it if
+you have a KGPE-D16 board, you will love it. The pre-built binaries can be
+found on [Dasharo documentation page](https://docs.dasharo.com/variants/asus_kgpe_d16/releases/)
+
+![](/img/dasharo-sygnet.svg)
+
+Besides coreboot improvements, the project charter also included upstreaming of
+the [flashrom patches](https://review.coreboot.org/c/flashrom/+/59713) for
+write protection and OTP memory management. These were especially intended for
+use with KGPE-D16 platform and the non-standard flashes to form a Static Root
+of Trust with immutable coreboot's bootblock and provide best security possible
+on this platform.
+
+The refreshed code for ASUS KGPE-D16 may be found on [Dasharo GitHub](https://github.com/Dasharo/coreboot/tree/asus_kgpe-d16/develop)
+and will be upstreamed to the main coreboot branch soon.
```

Comment:
@miczyg1 please mention Dasharo matrix where we are availabe for discussing and supporting the project.

Reply:
Added